usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+Instala�ao abortada.}
+
+message delete_xfer_file
+{Excluir ap�s a instala�ao}
+
+/* Called with: 			Example
+ *  $0 = set name			base
+ */
+message notarfile
+{O conjunto de lan�amento $0 nao existe.}
+
+message endtarok
+{Todos os conjuntos de distribui�ao selecionados foram descompactados com sucesso.}
+
+message endtar
+{Houve problemas ao descompactar conjuntos de distribui�ao.
+Sua instala�ao est� incompleta.
+
+Voc� selecionou %d conjuntos de distribui�ao. %d conjuntos nao puderam ser encontrados
+e %d foram ignorados ap�s a ocorr�ncia de um erro. Dos %d
+que foram tentados, %d descompactaram sem erros e %d com erros.
+
+Abortando a instala�ao. Verifique novamente sua fonte de distribui�ao
+e considere reinstalar os conjuntos do menu principal.}
+
+message abort_inst {Instala�ao abortada.}
+message abort_part {Particionamento abortado.}
+
+message abortinst
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Instala�ao abortada.
+}
+
+message abortupgr
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Atualiza�ao abortada.
+}
+
+message abortunpack
+{Descompactar conjuntos adicionais nao foi bem-sucedido. Voc� precisar�
+continuar manualmente ou escolher uma fonte diferente para conjuntos de lan�amento e tentar
+novamente.
+}
+
+message createfstab
+{TH� um grande problema! Nao � poss�vel criar /mnt/etc/fstab. Saindo!
+}
+
+
+message noetcfstab
+{Socorro! Nenhum /etc/fstab no disco de destino %s. Abortando atualiza�ao.
+}
+
+message badetcfstab
+{Socorro! Nao � poss�vel analisar /etc/fstab no disco de destino %s. Abortando atualiza�ao.
+}
+
+message X_oldexists
+{Nao consigo salvar %s/bin/X como %s/bin/X.old, porque o
+disco de destino j� tem um %s/bin/X.old. Corrija isso antes
+de continuar.
+
+Uma maneira � iniciar um shell no menu Utilit�rios, examinar o
+destino %s/bin/X e %s/bin/X.old. Se
+%s/bin/X.old for de uma atualiza�ao conclu�da, voc� pode rm -f
+%s/bin/X.old e reiniciar. Ou se %s/bin/X.old for de
+uma atualiza�ao recente e incompleta, voc� pode rm -f %s/bin/X e mv
+%s/bin/X.old para %s/bin/X
+
+Abortando a atualiza�ao.}
+
+message netnotup
+{Houve um problema na configura�ao da rede. Ou seu gateway
+ou seu servidor de nomes nao estava acess�vel por um ping. Voc� quer
+configurar sua rede novamente? ("Nao" permite que voc� continue de qualquer forma
+ou aborte o processo de instala�ao.)
+}
+
+message netnotup_continueanyway
+{Voc� gostaria de continuar o processo de instala�ao mesmo assim e assumir
+que a rede est� funcionando? ("Nao" aborta o processo de instala��o.)
+}
+
+message makedev
+{Criando n�s de dispositivos...
+}
+
+/* 
+* TODO
+*/
+/* Called with:				Example
+ *  $0 = device name			/dev/rwd0a
+ *  $1 = file system type		ffs
+ *  $2 = error return code form fsck	8
+ */
+message badfs
+{It appears that $0 is not a $1 file system or the fsck was
+not successful.  Try mounting it anyway?  (Error number $2.)
```

Comment:
I suppose this is where your WIP currently stops. Thank you for all the effort so far!
Hope I don't sound pedantic or picky with my suggestions. I just want to help you make the best damn BSD translation we can :)